### DISCUSSION TITLE
Can ticket be added to the scope?

### DISCUSSION THREAD

**Discussion by schophil** [Everduin94/better-commits#69] [Q&A] [answered]
At my company we made the decision to have commits like 'feat(t#icket): loremipsuim'. Can I configure better-commits to use the ticket as scope?

**Comment by Everduin94** [comment, marked as answer]
I created this issue: https://github.com/Everduin94/better-commits/issues/70

The idea is to allow users to have a configurable template so they can place whichever fields / separators as they see fit. :)

Although, it may take me a second to get around to this 😅
